<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CNetworkSetup</class>
 <widget class="QDialog" name="CNetworkSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QCheckBox::indicator {
     width: 30px;
     height: 30px;
}

QCheckBox::indicator:checked
{
	image: url(:images/checkbox_checked.png);
}
QCheckBox::indicator:unchecked
{
	image: url(:images/checkbox_normal.png);
}

QRadioButton::indicator {
     width: 30px;
     height: 30px;
}

QRadioButton::indicator:checked
{
	image: url(:images/radiobutton_checked.png);
}
QRadioButton::indicator:unchecked
{
	image: url(:images/radiobutton_normal.png);
}
</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>811</width>
     <height>631</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕코딩</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>LAN1</string>
   </property>
   <widget class="QRadioButton" name="lan1Dhcp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>자동으로 IP주소 받기</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">lan1ButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="lan1Static">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>다음 IP주소 사용</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">lan1ButtonGroup</string>
    </attribute>
   </widget>
   <widget class="IpEdit" name="lan1IP">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>300</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>IP 주소</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="IpEdit" name="lan1Subnetmask">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>300</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>서브넷마스크</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lan1Mac">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>420</y>
      <width>300</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>420</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MAC 주소</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>게이트웨이 주소</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="IpEdit" name="lan1Gateway">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>330</y>
      <width>300</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="lan1DefaultGateway">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>521</y>
      <width>201</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>기본 게이트웨이</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>230</y>
     <width>691</width>
     <height>641</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕코딩</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>LAN2</string>
   </property>
   <widget class="QRadioButton" name="lan2Dhcp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>자동으로 IP주소 받기</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">lan2ButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="lan2Static">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>다음 IP주소 사용</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">lan2ButtonGroup</string>
    </attribute>
   </widget>
   <widget class="IpEdit" name="lan2IP">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>300</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>IP 주소</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="IpEdit" name="lan2Subnetmask">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>250</y>
      <width>300</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>서브넷마스크</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lan2Mac">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>430</y>
      <width>300</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>430</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MAC 주소</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>게이트웨이 주소</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="IpEdit" name="lan2Gateway">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>340</y>
      <width>300</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="lan2NotUsed">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>사용안함</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">lan2ButtonGroup</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="lan2DefaultGateway">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>211</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕코딩</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>기본 게이트웨이</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>890</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕코딩</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>기본 DNS서버 주소</string>
   </property>
  </widget>
  <widget class="IpEdit" name="dnsSecondary">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>980</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕코딩</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="IpEdit" name="dnsPrimary">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>890</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕코딩</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>980</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕코딩</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>보조 DNS서버 주소</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>1470</x>
     <y>5</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕코딩</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>저장</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>1690</x>
     <y>5</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕코딩</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sysMange">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕코딩</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>시스템관리</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testMode">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>5</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕코딩</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>테스트모드</string>
   </property>
  </widget>
  <widget class="QPushButton" name="versionInfo">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>5</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕코딩</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>버전정보</string>
   </property>
  </widget>
  <widget class="QLabel" name="serialNoLabel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>1118</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕코딩</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>일련번호:</string>
   </property>
  </widget>
  <widget class="QLabel" name="serialNo">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>1118</y>
     <width>971</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕코딩</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>0000000000(0000-00-00 00:00:00) SS-1000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRoute">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>5</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕코딩</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>라우트</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnModel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕코딩</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>모델 변경</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHasuSetup">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>120</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕코딩</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>서버설정</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPingTest">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>5</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕코딩</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ping Test</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IpEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">ipedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="lan1ButtonGroup"/>
  <buttongroup name="lan2ButtonGroup"/>
 </buttongroups>
</ui>
